ui/app/models/pki/key.js

```diff
@@ -1,17 +1,40 @@
 import Model, { attr } from '@ember-data/model';
 import { inject as service } from '@ember/service';
 import { withFormFields } from 'vault/decorators/model-form-fields';
+import { withModelValidations } from 'vault/decorators/model-validations';
 
-@withFormFields(['keyId', 'keyName', 'keyType', 'keyBits'])
+const validations = {
+  type: [{ type: 'presence', message: 'Type is required.' }],
+};
+const displayFields = ['keyId', 'keyName', 'keyType', 'keyBits'];
+const formFieldGroups = [{ default: ['keyName', 'type'] }, { 'Key parameters': ['keyType', 'keyBits'] }];
+@withModelValidations(validations)
+@withFormFields(displayFields, formFieldGroups)
 export default class PkiKeyModel extends Model {
   @service secretMountPath;
 
-  @attr('boolean') isDefault;
```

Comment:
removed for now since we've decided to not surface this in the UI for now as this does not correlate to being the default issuer's key. there's potential the backend will ad a `is_default_issuer_key` param of some sort, but it's tbd